repositories/rhel-8.1.json

```diff
@@ -2,24 +2,24 @@
     "aarch64": [
         {
             "id": "baseos",
-            "baseurl": "http://download.devel.redhat.com/rhel-8/rel-eng/RHEL-8/latest-RHEL-8.1/compose/BaseOS/aarch64/os",
+            "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8/aarch64/baseos/os",
```

Comment:
Please include the minor version in these URLs. Otherwise, all of these will resolve to the latest RHEL 8.

`"https://cdn.redhat.com/content/dist/rhel8/8.1/aarch64/baseos/os"`

Reply:
This is fixed